idb/postgres/postgres_migrations_integration_test.go

```diff
@@ -0,0 +1,294 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/orlangure/gnomock"
+	"github.com/orlangure/gnomock/preset/postgres"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/algorand/go-algorand-sdk/encoding/json"
+	sdk_types "github.com/algorand/go-algorand-sdk/types"
+
+	"github.com/algorand/indexer/accounting"
+	"github.com/algorand/indexer/idb"
+	"github.com/algorand/indexer/importer"
+	"github.com/algorand/indexer/types"
+	"github.com/algorand/indexer/util/test"
+)
+
+func setupDb(t *testing.T) (*IndexerDb /*db*/, func() /*shutdownFunc*/) {
```

Comment:
Could you reuse `setupPostgres` instead? Having a single entrypoint for this would make #471 easier

Reply:
I was going to move it and other functions to `postgres_integration_test_common.go` and replace `setupPostgres` with it. Should I do it in this PR?

Reply:
No, that sounds good.